### PR TITLE
Pkg: remove the `init` command

### PR DESCRIPTION
**Pull request by KristofferC**
(cherry picked from commit 85b2ec4d4f945dad47f686ae9fb32ce04e8f659e)

### CODE REVIEW

**Comment by StefanKarpinski** [comment]
Thanks, @KristofferC!

File: docs/src/index.md [hunk 1]
```diff
@@ -491,7 +491,7 @@ total 8
 -rw-r--r-- 1 stefan staff 207 Jul  3 16:35 Manifest.toml
 -rw-r--r-- 1 stefan staff  56 Jul  3 16:35 Project.toml
 
-hell> cat Project.toml
+shell> cat Project.toml
```
Comment (KristofferC) [inline docs/src/index.md]: 👿 ➡️ 🐚

**Comment by StefanKarpinski** [comment]
@KristofferC: is this unlink error on Win64 a known issue? Doesn't seem to be related.

**Comment by StefanKarpinski** [comment]
I'm going to assume that failure is unrelated.

**Pull request merged**